config.lua

```diff
@@ -25,7 +25,7 @@ Config.Zones = {
 		Size  = { x = 1.5, y = 1.5, z = 1.0 },
```

Comment:
Comma is not the problem but declaration of an integer value on coordinates. 

```
Config.Zones = {

	MechanicActions = {
		Pos   = { x = 1003.2, y = -2314, z = 29.5},
		Size  = { x = 1.5, y = 1.5, z = 1.0 },
		Color = { r = 204, g = 204, b = 0 },
		Type  = 1
	}
}
```
-2314 is not a valid value for y coordinates.